elixir/apps/web/lib/web/controllers/home_controller.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Web.HomeController do
+  use Web, :controller
+  alias Domain.Accounts
+
+  def home(conn, _params) do
+    {accounts, conn} =
+      with {:ok, recent_account_ids, conn} <- Web.Auth.list_recent_account_ids(conn),
+           {:ok, accounts} <- Accounts.list_accounts_by_ids(recent_account_ids) do
+        conn =
+          Web.Auth.update_recent_account_ids(conn, fn _recent_account_ids ->
+            Enum.map(accounts, & &1.id)
+          end)
+
+        {accounts, conn}
+      else
+        _other -> {[], conn}
+      end
+
+    conn
+    |> put_layout(html: {Web.Layouts, :public})
+    |> render("home.html", accounts: accounts)
+  end
```

Comment:
I'm not sure if I have the wrong expectations or if this is not working as intended for me.  When running locally, I start at `http://localhost:13000/`, then type `firezone` in and click 'Go to Sign In page'.  I'm taken to the appropriate sign-in page and I successfully sign in.  After that I sign out and land back on the `firezone` account sign-in page.  I then manually navigate back to `http://localhost:13000/`, but I don't see the `firezone` account listed as one of the 'Recently used accounts'.  

Is that intended?

Reply:
@bmanifold yeah, when you "sign out" explicitly we remove that account ID from the list of recently used accounts. This is similar to how Gmail works - you will see your account in recent ones (even if you are not signed in currently) as long as you did not log out explicitly from them.

The design goal was to have a way to find a recently used/created account when you don't remember the slug/url, where there you probably would not sign out explicitly.

I'm totally open to changing it, but I guess there can be security considerations before we do that. For example: imagine that you are a contractor that works for multiple companies using Firezone and you don't want them to accidentally see that you had multiple recent accounts. We can workaround it by allowing erasing that list but it's just more features to build.

Reply:
I see where you're coming from.  The security concern you listed doesn't strike me as the most worrisome, but I won't say that it's something to completely ignore either.

I don't have super strong feelings on it, so maybe we can quickly get everyones feedback at standup and go from there.

Reply:
Just as a data point, I tried logging out explicitly with my google account and when I was asked to log back in it showed me by account that I had explicitly logged out of.